conduct.md

```diff
@@ -84,35 +86,37 @@ TODO: Write an introductory paragraph.
 
 ### Decisions
 
-TODO: Write an introductory paragraph.
+Decisions are the process by which team members collectively discuss their project ideas, define their goals, analyze problems, and make plans to achieve the goal. Teams must have a clearly defined process for making decisions before they start their projects so that they can effectively communicate with each other and make important decisions that enable them to write successful and sustainable software.
 
 #### Acceptable Conduct
 
-- TODO
-- TODO
-- TODO
+- Teams should finalize decisions when the team reaches a majority vote.
+- Teams should consider and discuss all possible ideas of each student before moving forward.
+- Teams should document how they reached the final decisions.
+- Teams should make space for quieter voices on the team to speak.
+- Students should not take final decisions personally.
 
 #### Unacceptable Conduct
 
-- TODO
-- TODO
-- TODO
+- Students should not talk over others or prevent others from submitting their ideas.
+- Students should not commit and push code that falls outside of the group's initial intent for a project.
+- Students should update documentation or code in ways that are consistent with the decisions made by the team.
```

Comment:
This last point doesn't follow the "should not" format like the rest of the unacceptable conducts.

Reply:
An easy fix would be to write something like "Students should not update documentation or code in ways that are inconsistent with the decisions made by the team". Other than that, this looks good!